.travis.yml

```diff
@@ -158,66 +158,19 @@ script:
 
   # ray tune tests
   - python python/ray/tune/test/dependency_test.py
-  - python -m pytest -v --durations=10 python/ray/tune/test/trial_runner_test.py
-  - python -m pytest -v --durations=10 python/ray/tune/test/trial_scheduler_test.py
-  - python -m pytest -v --durations=10 python/ray/tune/test/experiment_test.py
-  - python -m pytest -v --durations=10 python/ray/tune/test/tune_server_test.py
-  - python -m pytest -v --durations=10 python/ray/tune/test/ray_trial_executor_test.py
-  - python -m pytest -v --durations=10 python/ray/tune/test/automl_searcher_test.py
+  # `cluster_tests.py` runs on Jenkins, not Travis.
+  - python -m pytest -v --durations=30 --ignore=python/ray/tune/cluster_tests.py python/ray/tune/test
```

Comment:
I'm slightly surprised that this works because I thought `pytest` only picks up tests that have `test_` as the start of the name. But it seems to be working..